mlir/lib/Target/SPIRV/Deserialization/DeserializeOps.cpp

```diff
@@ -86,6 +86,13 @@ Value spirv::Deserializer::getValue(uint32_t id) {
   if (auto undef = getUndefType(id)) {
     return spirv::UndefOp::create(opBuilder, unknownLoc, undef);
   }
+  if (std::optional<spirv::GraphConstantARMOpMaterializationInfo>
+          graphConstantARMInfo = getGraphConstantARM(id)) {
+    IntegerAttr graphConstantID = graphConstantARMInfo->graphConstantID;
+    Type resultType = graphConstantARMInfo->resultType;
+    return opBuilder.create<spirv::GraphConstantARMOp>(unknownLoc, resultType,
```

Comment:
This is deprecated, use free create functions: https://discourse.llvm.org/t/psa-opty-create-now-with-100-more-tab-complete/87339